TOMP-API.yaml

```diff
@@ -594,6 +594,12 @@ paths:
             schema:
               $ref: "#/components/schemas/legEvent"
       responses:
+        '200':
+          description: operation successful
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/leg'
         '204':
```

Comment:
I think this suffices for the hotfix (which has to be released fast), but perhaps we should look into defining when a TO should use the 200 response with data or the 204 without.

Reply:
agree. But when operating locks etc this event supplies the assetAccessData...